packages/components/src/custom-select-control-v2/types.ts

```diff
@@ -187,4 +187,10 @@ export type CustomSelectItemProps = {
 	 * used if left `undefined`.
 	 */
 	children?: React.ReactNode;
+	/**
+	 * If true, the item will be disabled.
+	 *
```

Comment:
```suggestion
	 * If true, the item will be disabled.
	 *
	 * You will need to add your own styles (e.g. reduced opacity) to visually show that they are disabled.
```

or something like that

Reply:
Good point 👍